packages/database/prisma/schema.prisma

```diff
@@ -8,7 +8,213 @@ datasource db {
 }
 
 model Study {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  id                         Int                       @id
+  name                       String
+  //cpmsId                     String
+  status                     String
+  //statusId                   String
+  recordStatus               String
+  route                      String
+  irasId                     String?
+  protocolReferenceNumber    String?
+  sampleSize                 Int?
+  chiefInvestigatorFirstName String?
+  chiefInvestigatorLastName  String?
+  managingSpeciality         String
+  plannedOpeningDate         DateTime?
+  plannedClosureDate         DateTime?
+  actualOpeningDate          DateTime?
+  actualClosureDate          DateTime?
+  assessments                Assessment[]
+  organisations              StudyOrganisation[]
+  funders                    StudyFunder[]
+  evaluationCategory         StudyEvaluationCategory[]
+  createdAt                  DateTime                  @default(now())
+  updatedAt                  DateTime                  @updatedAt
+  isDeleted                  Boolean?                  @default(false)
+}
+
+model Organisation {
+  id            Int                 @id @default(autoincrement())
+  name          String              @unique
+  rtsIdentifier String              @unique
+  createdAt     DateTime            @default(now())
+  updatedAt     DateTime            @updatedAt
+  isDeleted     Boolean?            @default(false)
+  users         UserOrganisation[]
+  roles         OrganisationRole[]
+  studies       StudyOrganisation[]
+  fundedStudies StudyFunder[]
+
+  @@index([name])
+}
+
+model OrganisationRole {
+  id             Int                    @id @default(autoincrement())
+  organisation   Organisation           @relation(fields: [organisationId], references: [id])
+  organisationId Int
+  role           SysRefOrganisationRole @relation(fields: [roleId], references: [id])
+  roleId         Int
+  createdAt      DateTime               @default(now())
+  updatedAt      DateTime               @updatedAt
+  isDeleted      Boolean?               @default(false)
+
+  @@unique([organisationId, roleId])
+}
+
+model User {
+  id                    Int                @id @default(autoincrement())
+  emailAddress          String
+  identityGatewayId     String
+  registrationToken     String
+  registrationConfirmed Boolean            @default(false)
+  isDeleted             Boolean?           @default(false)
+  organisations         UserOrganisation[] @relation(name: "userOrganisation")
+  createdOrganisations  UserOrganisation[] @relation(name: "createdOrganisations")
+  updatedOrganisations  UserOrganisation[] @relation(name: "updatedOrganisations")
+  roles                 UserRole[]         @relation(name: "userRoles")
+  createdRoles          UserRole[]         @relation(name: "createdRoles")
```

Comment:
Same here.. not sure we need created/updated?

Reply:
These are the opposite relations for the `createdBy` and `updatedBy` fields on `UserRole`.